packages/elements/src/components/ino-datepicker/ino-datepicker.tsx

```diff
@@ -129,12 +130,17 @@ export class Datepicker implements ComponentInterface {
     this.create();
   }
 
+  @Watch('disabled')
+  disabledChanged() {
+    this.dispose();
+  }
```

Comment:
There are two use-cases that need to be addressed here:
- Datepicker will be disabled -> Datepicker will be disposed
- Datepicker will be enabled -> Datepicker will be created

```suggestion
  @Watch('disabled')
  disabledChanged(newValue: boolean) {
  	if(newValue) {
  		this.dispose();
  	} else {
  		this.create();
    }
  }
```

Reply:
I agree, thanks for pointing it out!